### PR TITLE
clean and fixes

### PR DESCRIPTION
**Pull request by Ali-Razmjoo**
some fixes

### CODE REVIEW

File: api/server.py [hunk 1]
```diff
@@ -1210,31 +1247,17 @@ def start_api_server():
...
-
-        "api_access_log_filename":
-            my_api_configuration["api_access_log"]["filename"],
-
-        "api_access_without_key":
-            api_access_without_key,
-
+        "api_access_key": api_access_key,
+        "api_client_white_list": my_api_configuration["api_client_white_list"]["enabled"],
+        "api_client_white_list_ips": my_api_configuration["api_client_white_list"]["ips"],
```
Comment (dhirensr) [inline api/server.py:1257]: this is breaking the lint ,line too long (113 > 80 characters)

File: core/alert.py [hunk 2]
```diff
@@ -24,125 +23,29 @@ def is_not_run_from_api():
...
-
-def input_msg(content):
-    """
-    build the input message to get input from users
+    build the info message, log the message in
+    database if requested, rewrite the thread temporary file
 
     Args:
-        content: content of the message
+        content: content of the messagswe
```
Comment (dhirensr) [inline core/alert.py:32]: typo

File: api/server.py (hunk 1 above)
Comment (Ali-Razmjoo) [reply api/server.py:1252]: I reconfigure flake8 to 120 chars now.

File: errors [hunk 3]
```diff
@@ -0,0 +1,493 @@
+./core/color.py:22:1: C901 'color_cmd' is too complex (21)
```
Comment (ChakshuGupta) [inline errors:1]: Why do we need this file?

File: lib/ip2location/IP2Location.py [hunk 4]
```diff
@@ -329,7 +351,7 @@ def calc_off(what, mid):
         return rec
 
     def __iter__(self):
-        low, high = 0, self._ipv4dbcount
+        low, high = 0, self._ipv4dbcounts
```
Comment (ChakshuGupta) [inline lib/ip2location/IP2Location.py:354]: Variable defined in the class above is  `_ipv4dbcount` not `_ipv4dbcounts`

File: errors (hunk 3 above)
Comment (Ali-Razmjoo) [reply errors:1]: mistake...

File: core/alert.py [hunk 5]
```diff
@@ -24,125 +22,29 @@ def is_not_run_from_api():
...
-                    "utf8"
-                )
-            )
-            sys.stdout.flush()
+    sys.stdout.buffer.write(
+        bytes(
+            color.color_cmd("yellow")
+            + "[+] [{0}] ".format(now())
+            + color.color_cmd("green")
+            + content
```
Comment (dhirensr) [inline core/alert.py:41]: this would be perfect if we are just supporting python3, then I guess we can remove version function too from compatible

Comment (Ali-Razmjoo) [reply core/alert.py:41]: it's already only python3...

**Pull request merged**